### PR TITLE
blog post about new web graphics

### PR DESCRIPTION
**Pull request by couet**
New blog post anouncing the new graphics default.

### CODE REVIEW

**Comment by linev** [comment]
Probably some conclusion is missing. Like: 

We encourage all users to try this functionality and give us the feedback!

**Comment by bellenot** [comment]
I'll take a look and let you know (maybe a couple of details here and there...)

File: blog/_posts/2023-06-05-web-based-graphics-default.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+After a long period of development and testing we decided to switch to the web-based `TCanvas`
+implementation by default in the ROOT master version. It has been present in the ROOT for a
+while (since 2017) and used already in the web-based TBrowser, which you have probably seen already.
+
+What has changed ? Now when starting a ROOT session and displaying any object in `TCanvas`,
+the default system web browser will start and the object will be drawn there using the JavaScript
+ROOT functionality. The look and feel for basic objects, like histograms and graphs, will not
+change much – all the drawing options and styles are supported as in the original graphics.
+You can compare the two following screen shots made with the same macro – one is original
+ROOT graphics, other is web-based one. Try to guess which one where?
```
Comment (bellenot) [inline blog/_posts/2023-06-05-web-based-graphics-default.md:16]:
```suggestion
ROOT graphics, other is web-based one.
```
I think `Try to guess which one where?` is superfluous

File: blog/_posts/2023-06-05-web-based-graphics-default.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ 5.  The Web is a very natural way for implementing remote displays. Instead of struggling with
+     remote X11 one can just use a local web browser accessing remote applications through
+     http. To ease configuration of `ssh` tunnels, we provide a simple `rootssh` utility which
+     fully automates the configuration of such tunnels.
+ 6.  One can use QWebEngine (Qt5 and Qt6) to implement a fully local display without any http
+     server in-between. This allows embedding any kind of ROOT web-based widgets into Qt
+     applications on all platforms.
+
+What about image production in batch?
+For the moment we keep old functionality, like when running ROOT with the `-b` flag, for image
```
Comment (bellenot) [inline blog/_posts/2023-06-05-web-based-graphics-default.md:42]:
```suggestion
For the moment we keep the old functionality, like when running ROOT with the `-b` flag, for image
```

File: blog/_posts/2023-06-05-web-based-graphics-default.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+production. Web-based canvas will be used for PNG/JPEG/SVG images creation by adding `--web`
+flag when running ROOT. While image generation involves running of web browsers in headless
```
Comment (bellenot) [inline blog/_posts/2023-06-05-web-based-graphics-default.md:44]:
```suggestion
flag when running ROOT. While image generation involves running web browsers in headless
```

**Review by bellenot — APPROVED** [review]
Several small details, but overall it looks OK

**Review by linev — APPROVED** [review]

**Pull request merged**